feg/gateway/docker/docker-compose.yml

```diff
@@ -112,3 +112,7 @@ services:
       sh -c "/usr/local/bin/generate_service_config.py --service=redis --template=redis &&
              /usr/bin/redis-server /var/opt/magma/tmp/redis.conf --daemonize no &&
              /usr/bin/redis-cli shutdown"
+
+volumes:
```

Comment:
We can remove these volumes as they're only intended for development environment. In the new prod_env file, you can set:
```
CERTS_VOLUME=/var/opt/magma/certs
CONFIGS_VOLUME=/var/opt/magma/configs
```

Reply:
fixed